internal/config/gofer/origin.go

```diff
@@ -23,98 +23,110 @@ import (
 	"github.com/makerdao/oracle-suite/pkg/gofer/origins"
 )
 
-type apiKeyParams struct {
-	APIKey string `json:"apiKey"`
+type originParams struct {
+	APIKey        string                    `json:"apiKey"`
+	Contracts     origins.ContractAddresses `json:"contracts"`
+	SymbolAliases origins.SymbolAliases     `json:"symbolAliases"`
```

Comment:
I think this should be handled by every origin separately.

Reply:
This you mean `APIKey` only or all options ?